src/views/Alteration.vue

```diff
@@ -19,10 +19,6 @@
         <ShareStructures class="mt-10" />
 
         <Articles class="mt-10" />
-
-        <AgreementType class="mt-10" v-if="false" />
-
-        <Detail class="mt-10" v-if="false" />
```

Comment:
Where did these go?

Reply:
Removed them as they were not being used.

Reply:
OK!

Oh, haha, I see the "v-if" now :)